go.mod

```diff
@@ -138,6 +138,5 @@ replace (
 	github.com/docker/scout-cli => github.com/docker/scout-cli v1.18.4
 	github.com/moby/buildkit => github.com/moby/buildkit v0.25.0
 	github.com/moby/moby => github.com/moby/moby v28.4.0+incompatible
+	github.com/docker/mcp-gateway => github.com/docker/mcp-gateway v0.22.0
```

Comment:
I'll have a look at removing the replace rules in a follow-up (looks like dependencies were currently pinned, and shouldn't be, but didn't want to put those changes in the same PR)